register.py

```diff
@@ -141,11 +152,12 @@ def request_token(pool, resource):
 
     print("Token request approved!")
 
-    token_dir = htcondor.param["SEC_TOKEN_DIRECTORY"]
+    global TOKEN_DIR
+    if TOKEN_DIR == "":
+        TOKEN_DIR = htcondor.param["SEC_TOKEN_DIRECTORY"]
     token_name = "50-{}-{}-registration".format(alias, resource)
-    token_path = os.path.join(token_dir, token_name)
-
-    logger.debug("Writing token to disk (in {})".format(token_dir))
+    token_path = '/'.join(os.path.join(TOKEN_DIR, token_name).split('\\'))
+    logger.debug("Writing token to disk (in {})".format(TOKEN_DIR))
```

Comment:
```suggestion
    logger.debug("Writing token to disk (in {})".format(token_path))
```

It's also worth noting that this message will be incorrect with the above value of `TOKEN_DIR` since the user is mounting `$PWD/tokens` to the path where the script is going to drop the requested tokens whereas `TOKENS_DIR` passed by `--dir` (or `--local-dir`) will only be `$PWD`